src/io.jl

```diff
@@ -135,11 +136,8 @@ function get_plant_data(coordpath = GENCOORDPATH)
             (df[i, Symbol("Latitude")], df[i, Symbol("Longitude")])
         )
         vs = Real[]
-        for s in [
-            Symbol("Grid Voltage (kV)"),
-            Symbol("Grid Voltage 2 (kV)"),
-            Symbol("Grid Voltage 3 (kV)")
-        ]
+
+        for s in [Symbol("Grid Voltage (kV)"), Symbol("Grid Voltage 2 (kV)"), Symbol("Grid Voltage 3 (kV)")]
```

Comment:
If these are all the column names

```suggestion
        for s in names(df)
```
otherwise
```suggestion
        for s in keys(column_types)
```

Reply:
I believe `df` has other columns, so we can't simply call `names` or `keys`.

Reply:
we can use `column_types` because we know that has these names 
https://github.com/invenia/SyntheticGrids.jl/pull/17/files#diff-b5ddd5a415e261ce7d1b14e8bb8f9dfaR127